src/lib/yang/util.lua

```diff
@@ -129,6 +129,20 @@ function memoize(f, max_occupancy)
    end
 end
 
+local function gmtime ()
+   local now = os.time()
+   local utcdate = os.date("!*t", now)
+   local localdate = os.date("*t", now)
+   localdate.isdst = false
+   local timediff = os.difftime(os.time(utcdate), os.time(localdate))
+   return now + timediff
+end
+
+function iso_8601 (time)
```

Comment:
Can we rename this function?  `format_date_as_iso_8601` maybe -- too long but at least it includes the word `date`.  Dunno, other ideas possible, otherwise it's a bit hard to read.